src/main/java/org/owasp/html/CssSchema.java

```diff
@@ -423,6 +423,8 @@ Property forKey(String propertyName) {
         "auto", "inherit", "none");
     Set<String> overflowLiterals0 = Set.of(
         "auto", "hidden", "inherit", "scroll", "visible");
+    Set<String> overflowWrapLiterals0 = Set.of(
+        "normal", "break-word", "anywhere", "inherit");
```

Comment:
What about other valid values such as `"initial" , "revert" , "revert-layer", "unset"` ((listed as "global values" on https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap?

Reply:
I think revert and revert-layer were added before the last time this property list was thoroughly revamped.  I don't quite understand the semantics, but I would worry about adding it everywhere that `inherit` appears in case it allows working around visual containment and event capturing measures like clipping, position, and overflow.